source/frontend/src/features/properties/map/acquisition/add/__snapshots__/AddAcquisitionContainer.test.tsx.snap

```diff
@@ -1509,6 +1509,7 @@ exports[`AddAcquisitionContainer component renders as expected 1`] = `
 .c0 {
   position: -webkit-sticky;
   position: sticky;
+  padding-top: 2rem;
```

Comment:
This is intentional right? affecting the add acquisition/activity/research screens?

Reply:
Yes, I saw there was no padding on the actions at the top, causing the buttons to almost clip the top content